src/attributes.jl

```diff
@@ -440,6 +440,15 @@ given.
 """
 struct Silent <: AbstractOptimizerAttribute end
 
+"""
+    TimeLimitSec
+
+An optimizer attribute for setting a time limit to an optimization. When `set`
+to `nothing`, it deactivate the solver time limit. The default value is
```

Comment:
I'm confused about which solvers have a default finite time limit. Which are they?

Reply:
By "it deactivate the solver time limit" I meant it set it back to default. For CPLEX default value is 1e+75.